upload.prx.org/pipeline.yml

```diff
@@ -2,7 +2,7 @@ AWSTemplateFormatVersion: "2010-09-09"
 Description: >
   Creates a CodePipeline that is used to handle CI/CD for the upload lambda
 Parameters:
-  OpsDebugMessagesSNSTopicArn:
+  OpsWarnMessagesSNSTopicArn:
```

Comment:
It doesn't look like this ever gets used in the pipeline template. It's fine to pass in extra params, but it's unlikely a build error would ever get sent to **warn**, so in this case it may be worth taking out.